civictechprojects/helpers/context_preload.py

```diff
@@ -111,6 +111,17 @@ def create_event_preload(context, request):
     context['description'] = 'Create event page'
     return context
 
+def create_group_preload(context, request):
+    context = default_preload(context, request)
+    context['title'] = 'Create an group | DemocracyLab'
```

Comment:
```suggestion
    context['title'] = 'Create a Group | DemocracyLab'
```

this is an old mistake, not yours, but please fix this inconsistency with casing and grammar between group and project.